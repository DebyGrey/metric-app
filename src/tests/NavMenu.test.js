import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Navmenu from '../components/NavMenu';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <Navmenu />
      </BrowserRouter>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
