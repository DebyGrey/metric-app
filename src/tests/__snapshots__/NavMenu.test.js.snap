// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<header>
  <nav
    className="container d-flex-space-be"
  >
    <div
      className="logo"
    >
      <a
        aria-current="page"
        className="active"
        href="/"
        onClick={[Function]}
      >
        CRYPSHARE
      </a>
    </div>
    <ul>
      <li>
        <a
          className=""
          href="/about"
          onClick={[Function]}
        >
          About
        </a>
      </li>
    </ul>
  </nav>
</header>
`;
